lib/factory_girl/strategy/stub.rb

```diff
@@ -75,6 +75,28 @@ def created_at
             end
           end
         end
+
+        responds_to_updated_at = result_instance.respond_to?(:updated_at)
+        updated_at_missing_default = responds_to_updated_at && !result_instance.updated_at
+        result_instance_missing_updated_at = !responds_to_updated_at
+
+        if updated_at_missing_default || result_instance_missing_updated_at
```

Comment:
Nice question. I obviously kept the behaviour consistent with created_at behaviour. Let me know what would you prefer. We would have to make them consistent anyway.